src/OrchardCore.Modules/OrchardCore.Settings/Views/Settings.Edit.cshtml

```diff
@@ -16,6 +16,15 @@
     <span class="hint">@T["Enter the fully qualified base URL of the web site."]</span>
 </fieldset>
 
+<div class="row">
+    <fieldset class="col-xl-6 form-group" asp-validation-class-for="PageSize">
+        <label asp-for="PageSize">@T["Page Size"]</label>
+        <input asp-for="PageSize" type="number" class="form-control" />
+        <span asp-validation-for="PageSize"></span>
+        <span class="hint">@T["The Page size that will be used by the Pagers in the Admin Panel of the web site."]</span>
```

Comment:
The default page size. FIX maj typos. If you want to emphasize use bold of italic.